src/BinSkim.Rules/CryptoErrors.cs

```diff
@@ -10,6 +10,8 @@ namespace Microsoft.CodeAnalysis.IL.Rules
     /// </summary>
     public enum CryptoError : uint
     {
+        // Whenever this file is refreshed,
```

Comment:
Nit: suggest "modified" instead of "refreshed."

Reply:
fixed